src/modules/reporting/containers/reporting-dispute-markets.js

```diff
@@ -33,9 +34,11 @@ const mapStateToProps = (state, { history }) => {
     navigateToAccountDepositHandler: () => history.push(makePath(ACCOUNT_DEPOSIT)),
     outcomes: disputeOutcomes,
     account: loginAccount.address,
-    isForking: state.universe.forkEndTime && state.universe.forkEndTime !== '0',
+    isForking: state.universe.isForking,
     forkEndTime: state.universe.forkEndTime,
     currentTime: state.blockchain.currentAugurTimestamp,
+    isForkingMarketFinalized: state.universe.isForkingMarketFinalized,
+    forkingMarket: state.universe.forkingMarket,
   })
 }
```

Comment:
I see isForking is used in reporting-header but not used in reporting-dispute-markets. Inject isForking in reporting-header.js component container.

iiuc these changes are needed here.